src/js/views/maps/feature-tiles-layer.js

```diff
@@ -0,0 +1,236 @@
+/*jslint nomen: true */
+/*globals define */
+
+define(function (require, exports, module) {
+  'use strict';
+
+  var $ = require('jquery');
+  var _ = require('lib/lodash');
+  var Backbone = require('backbone');
+  var L = require('lib/leaflet/leaflet.tilejson');
+  var Promise = require('lib/bluebird');
+
+  var api = require('api');
+  var util = require('util');
+
+  // We reuse the cartodb layer templates
+  var infoTemplate = require('text!templates/cartodb-info.html');
+  var tooltipTemplate = require('text!templates/cartodb-tooltip.html');
+  var template = require('text!templates/projects/layerControl.html');
+
+  // Render an infowindow for a selected map item
+  var ItemView = Backbone.View.extend({
```

Comment:
This is basically a close cousin of `cartodb-layer.js`, so I've kept the same file structure and general workings for clarity. They're both used only in `js/views/surveys/multi.js`.

I think we could actually factor the `ItemView` out of both the cartodb and feature layer files into a single separate file. The only difference between the `ItemView` in the two files is the removal of a couple of lines in one, and it's probably appropriate for both, so we can consolidate.